### DISCUSSION TITLE
Release a new version?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#2904] [General]
When will Termux 0.119.0 release, I'm curious about it

**Comment by 2096779623** [comment]
> There is still work that needs to be done before a release can be made. Also need to merge some pull requests.

**Comment by agnostic-apollo** [comment]
Likely never.

**Reply by Maxython** [reply]
bruh ._.

**Reply by agnostic-apollo** [reply]
What? Termux now follows semantic versioning so **technically** its highly unlikely that `0.119` would ever get released, at least by me. ;)

Also with the amount of work, *who* knows...

**Reply by 2096779623** [reply]
0.120?

**Reply by agnostic-apollo** [reply]
Nope. Likely `0.119.0`.

https://semver.org

**Reply by agnostic-apollo** [reply]
One would have to remove checks, otherwise builds will fail and releases will automatically get deleted.

https://github.com/termux/termux-app/blob/21134078/app/build.gradle#L155

https://github.com/termux/termux-app/blob/21134078/.github/workflows/debug_build.yml#L40

https://github.com/termux/termux-app/blob/21134078/.github/workflows/attach_debug_apks_to_release.yml#L39

**Comment by luisdavim** [comment]
So no release planned for any time soon? Any specific issues that are blocking a new release? (Regardless of the version number)

**Reply by agnostic-apollo** [reply]
There a few things on my todo list that need to completed before a release can be made, including merging some pull requests.

**Comment by luisdavim** [comment]
Any updates on this front?

**Reply by agnostic-apollo** [reply]
Trying to target end of may. Let's see.

**Reply by luisdavim** [reply]
Awesome, thanks. That's great news.

**Reply by luisdavim** [reply]
Almost there? ;)

**Reply by agnostic-apollo** [reply]
Other stuff came up, working on site for termux docs and termux-exec stuff. After that a pre-release is planned before major release since work is still left.

**Reply by luisdavim** [reply]
Thanks for the update

**Comment by babaric-dev** [comment]
Can we have a (pre)release before the end of 2023? :)
